.github/workflows/ci.yaml

```diff
@@ -246,13 +246,24 @@ jobs:
           [
             "auth-client-test.ts",
             "billing-test.ts",
+            "card-endpoints-test.ts",
+            "card-source-endpoints-test.ts",
             "index-query-engine-test.ts",
             "index-writer-test.ts",
             "indexing-test.ts",
             "loader-test.ts",
             "module-syntax-test.ts",
+            "permissions/permission-checker-test.ts",
             "queue-test.ts",
+            "realm-endpoints/directory-test.ts",
+            "realm-endpoints/info-test.ts",
+            "realm-endpoints/mtimes-test.ts",
+            "realm-endpoints/permissions-test.ts",
+            "realm-endpoints/search-test.ts",
+            "realm-endpoints/user-test.ts",
             "realm-endpoints-test.ts",
```

Comment:
@lukemelia we are still running this test--do we still need to now that it's been split out?

Reply:
Yeah what's left is what I didn't extract into other files. It's a bit of a mish mash.